roles/hci_prepare/tasks/phase1.yml

```diff
@@ -60,6 +60,7 @@
           - op: replace
             path: /spec/services
             value:
+              - bootstrap
```

Comment:
phase-2 also chnage services:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/hci_prepare/tasks/phase2.yml#L95
But since bootstrap will run on phase-1, It won't be a problem. Just a matter of making things in sync.

Reply:
I have checked how it's done in the VA repo and that one is not getting called in the second deploy. So I think it's fine like this.